### PR TITLE
Document recent changes in NEWS and CHANGES

### PR DESCRIPTION
**Pull request by levitte**
More should be added there

### CODE REVIEW

**Comment by levitte** [comment]
Seriously, we're *really* *not* *treating* *these* *files* *well*, *at* *all*

File: CHANGES [hunk 1]
```diff
@@ -9,6 +9,14 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Introduced new error raising macros, ERR_raise() and ERR_raise_data(),
+     where the former acts as a replacement for ERR_put_error(), and the
+     latter replaces the combination ERR_put_error()+ERR_add_error_data().
+     ERR_raise_data() adds more flexibility by taking a format strings and
```
Comment (InfoHunter) [inline CHANGES]: should be 'by taking a format string' instead of 'strings'

**Review by InfoHunter — CHANGES_REQUESTED** [review]

File: CHANGES (hunk 1 above)
Comment (levitte) [reply CHANGES]: Fixed

**Review by InfoHunter — APPROVED** [review]
Looks good to me

**Comment by levitte** [comment]
Merged.

faea3bd1339ac1029ab2bc746dfb7c891366d653 Document recent changes in NEWS and CHANGES

**Pull request closed without merging**